src/components/select/select.jsx

```diff
@@ -21,7 +21,7 @@ import useMounted from '../../hooks/use-mounted.js';
 
 const Select = forwardRef(function Select(props, ref) {
   const [showOptions, setShowOptions] = useState(false);
-  const [value, setValue] = useState(props.value);
+  const [value, setValue] = useState(props.value || null);
```

Comment:
It might not matter, so feel free to tell me if it doesn't, but remember that an empty string is also falsey in javascript. Is it okay `props.value` changes from an empty string to `null`?

If not we can either use the nullish coalesce (`props.value ?? null`) or just a terenary here

Reply:
Good point, I do think we need to worry about that. I'll update!

Reply:
Fun fact, we can't currently use nullish coalescing in MDS. 🙃